applications/nrf5340_audio/src/bluetooth/bt_content_control/bt_content_ctrl.c

```diff
@@ -86,7 +86,7 @@ int bt_content_ctrl_conn_disconnected(struct bt_conn *conn)
 
 	if (IS_ENABLED(CONFIG_BT_MCC)) {
 		ret = bt_content_ctrl_media_conn_disconnected(conn);
-		if (ret) {
+		if (ret && ret != -ESRCH) {
```

Comment:
May I know in which condition would get -ESRCH?
And would that cause issue ?

Reply:
This should not cause an issue as the returned index is < 0 and hence logs a warning in the disconnect. The -ESRCH is then passed to the disconnect caller and as it is not 0 it errors. Finally the disconnect returns to the event handler where it again logs an error, the event handler does not return any value, and then the disconnect call-back finishes cleaning up.

Reply:
Can you add a comment to why `-ESRCH` is not an error? gWacey explanation sounds right, but I don't think the code is self explanatory on this one

Reply:
-ESRCH is returned when we try to do a disconnect on a conn pointer that doesn't exist in the MCS overview since security failed before it even got to the discovery part. I will add a comment